Draft of release highlights (#9742)
PrefectHQ/prefect: RELEASE-NOTES.md

# Prefect Release Notes

## Release 2.10.11

### Interactive Deployments and Work Pool Wizard 🧙
![Screenshot 2023-05-25 at 1 02 48 PM](https://github.com/PrefectHQ/prefect/assets/12350579/c861b8dd-2dbb-4cfa-82f9-69008714f9fe)

This release introduces enhancements aimed at simplifying deployment and work pool creation.

> Firstly, the `prefect deploy` command has been upgraded to provide interactive prompts for deployment names and work pool selections. If you don't provide a deployment name via the CLI or a deployment.yaml file, Prefect will prompt you to do so. Furthermore, if a work pool name isn't specified, Prefect will guide you through the available work pools for your workspace. This feature aims to make deployments more approachable, even for first-time users, requiring just an entrypoint to a flow to get started.

Reviewer: ```suggestion
Firstly, the `prefect deploy` command has been upgraded to provide interactive prompts for deployment names and work pool selections. If you don't provide a deployment name via the CLI or a `deployment.yaml` file, Prefect will prompt you to do so. Furthermore, if a work pool name isn't specified, Prefect will guide you through the available work pools for your workspace. This feature aims to make deployments more approachable, even for first-time users, requiring just an entrypoint to a flow to get started.
```

I'm unsure if it makes more sense with our release notes cadence to say "you will be prompted/guided" versus "Prefect will prompt/guide you". @billpalombi would know better than I

Author: I changed from 'Prefect' to 'the CLI.' Let me know if you think that reads better.

Reviewer 2: We have not been consistent on active vs passive voice. This reads well to me.

Secondly, we've added a work pool creation wizard to streamline the process and spotlight various infrastructure types. The wizard will walk you through the essentials: basic work pool info, infrastructure type, and infrastructure configuration. The infrastructure type step will present you with a list of available infrastructure types, each with an icon and a description.

> 

Reviewer: A UI screenshot or two here would be super

Author: Added!

# Prefect Release Notes

## Release 2.10.11

### Interactive Deployments and Work Pool Wizard 🧙

> This release introduces enhancements aimed at simplifying deployment and work pool creation.

Reviewer 2: ```suggestion
This release simplifies deployment and work pool creation.
```

> ![Screenshot 2023-05-25 at 1 02 48 PM](https://github.com/PrefectHQ/prefect/assets/12350579/c861b8dd-2dbb-4cfa-82f9-69008714f9fe)

Reviewer 3: Make sure to update this alt-text to something relevant

> Firstly, the `prefect deploy` command has been upgraded to provide interactive prompts for deployment names and work pool selections. If you don't provide a deployment name via the CLI or a `deployment.yaml` file, the CLI will prompt you to do so. Furthermore, if a work pool name isn't specified, the CLI will guide you through the available work pools for your workspace. This feature aims to make deployments more approachable, even for first-time users, requiring just an entrypoint to a flow to get started.

Reviewer 2: ```suggestion
Firstly, the `prefect deploy` command has been upgraded to provide interactive prompts for deployment names and work pool selections. If you don't provide a deployment name via the CLI or a `deployment.yaml` file, the CLI will prompt you to do so. Furthermore, if a work pool name isn't specified, the CLI will guide you through the available work pools for your workspace. This feature aims to make deployments more approachable, especially for first-time users, requiring just an entrypoint to a flow to get started.
```
